docker/lr-metrics/Dockerfile

```diff
@@ -30,6 +31,9 @@ RUN git clone https://github.com/broadinstitute/gatk.git -b kvg_pbeap \
 # install picard
 RUN wget -O /usr/local/bin/picard.jar https://github.com/broadinstitute/picard/releases/download/2.22.1/picard.jar
 
+# install gsutil
+RUN curl -sSL https://sdk.cloud.google.com | bash
```

Comment:
Unrelated to this command, just a convention in this repo.

We usually bump the version in the companion make file when the Dockerfile/environment file are changed.

Reply:
Reverted.